core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java

```diff
@@ -412,8 +412,22 @@ private void setMailProps() {
         DATAINFO.setProperty("mail.smtps.auth", DATAINFO.getProperty("mailSmtpsAuth"));
         DATAINFO.setProperty("mail.smtps.starttls.enable", DATAINFO.getProperty("mailSmtpsStarttls.enable"));
         DATAINFO.setProperty("mail.smtp.connectiontimeout", DATAINFO.getProperty("mailSmtpConnectionTimeout"));
-        DATAINFO.setProperty("mail.smtp.starttls.required", DATAINFO.getProperty("mailSmtpStarttls.required"));
-        DATAINFO.setProperty("mail.smtp.ssl.protocols", DATAINFO.getProperty("mailSmtpSslProtocols"));
+        /**
+         * As Microsoft are currently deprecating TLS versions 1.0 and 1.1
+         * set default value
+         */
+        if(DATAINFO.getProperty("mailSmtpStarttls.required") == null) {
+        	DATAINFO.setProperty("mail.smtp.starttls.required", "true");
+        }else {
+        	DATAINFO.setProperty("mail.smtp.starttls.required", DATAINFO.getProperty("mailSmtpStarttls.required"));
+        }
+        
+        if(DATAINFO.getProperty("mailSmtpSslProtocols") == null) {
+        	DATAINFO.setProperty("mail.smtp.ssl.protocols", "TLSv1.2");
+        }else {
+        	DATAINFO.setProperty("mail.smtp.ssl.protocols", DATAINFO.getProperty("mailSmtpSslProtocols"));
```

Comment:
I don't know if all our customers would be able to upgrade to TLSv1.2. I think we should set these properties only if the customer explicitly sets them. Otherwise, we should skip setting these properties.

Reply:
changed -- we should set these properties only if the customer explicitly sets them. Otherwise, we should skip setting these properties.